atlasdb-client/src/integrationInput/java/com/palantir/atlasdb/table/description/generated/SchemaApiTestV2Table.java

```diff
@@ -0,0 +1,271 @@
+
+    private final Transaction t;
+
+    private final TableReference tableRef;
+
+    private SchemaApiTestV2Table(Transaction t, Namespace namespace) {
+        this.tableRef = TableReference.create(namespace, rawTableName);
+        this.t = t;
+    }
+
+    public static SchemaApiTestV2Table of(Transaction t, Namespace namespace) {
+        return new SchemaApiTestV2Table(t, namespace);
+    }
+
+    public static String getRawTableName() {
+        return rawTableName;
+    }
+
+    public TableReference getTableRef() {
+        return tableRef;
+    }
+
+    public String getTableName() {
+        return tableRef.getQualifiedName();
+    }
+
+    public Namespace getNamespace() {
+        return tableRef.getNamespace();
+    }
+
+    /**
+     * Returns the value for column Column1 and specified row components. */
+    public Optional<Long> getColumn1(String component1) {
+        SchemaApiTestTable.SchemaApiTestRow row = SchemaApiTestTable.SchemaApiTestRow.of(component1);
+        byte[] bytes = row.persistToBytes();
+        ColumnSelection colSelection = 
+                ColumnSelection.create(ImmutableList.of(PtBytes.toCachedBytes("c")));
+        RowResult<byte[]> rowResult = t.getRows(tableRef, ImmutableSet.of(bytes), colSelection).get(bytes);
+        if (rowResult == null) {
+             return Optional.empty();
+        } else {
+             return Optional.of(SchemaApiTestTable.SchemaApiTestRowResult.of(rowResult).getColumn1());
+        }
+    }
+
+    /**
+     * Returns a mapping from row keys to value at column Column1.
+     * As the values are all loaded in memory, do not use for large amounts of data.
+     * If the column does not exist for a key, the entry will be omitted from the map. */
+    public Map<String, Long> getColumn1(Iterable<String> rowKeys) {
+        ColumnSelection colSelection = 
+                 ColumnSelection.create(ImmutableList.of(PtBytes.toCachedBytes("c")));
+        List<SchemaApiTestTable.SchemaApiTestRow> rows = Lists
+                .newArrayList(rowKeys)
+                .stream()
+                .map(SchemaApiTestTable.SchemaApiTestRow::of)
+                .collect(Collectors.toList());
+        SortedMap<byte[], RowResult<byte[]>> results = t.getRows(tableRef, Persistables.persistAll(rows), colSelection);
+        return results
+                .values()
+                .stream()
+                .map(entry -> SchemaApiTestTable.SchemaApiTestRowResult.of(entry))
+                .collect(Collectors.toMap(
+                     entry -> entry.getRowName().getComponent1(), 
+                     SchemaApiTestTable.SchemaApiTestRowResult::getColumn1));
+    }
+
+    /**
+     * Returns a mapping from all the row keys to their value at column Column1
+     * (if that column exists for the row-key). As the values are all loaded in memory,
+     * do not use for large amounts of data. */
+    public Map<String, Long> getAllColumn1() {
```

Comment:
i'm a little dubious of even providing this method. I can't think of when you'd really want to get all the rows in an unpaged manner, and when you do it's probably worth forcing you to use `getSmallRange` so you're sure of what you're doing